module/gui/_pyLoadGui.py

```diff
@@ -0,0 +1,625 @@
+    statePackageDockNode = mainWindowNode.toElement().elementsByTagName("statePackageDock").item(0)
+    stateLinkDockNode = mainWindowNode.toElement().elementsByTagName("stateLinkDock").item(0)
+    visibilitySpeedLimitNode = mainWindowNode.toElement().elementsByTagName("visibilitySpeedLimit").item(0)
+    visibilityMaxParallelDownloadsNode = mainWindowNode.toElement().elementsByTagName("visibilityMaxParallelDownloads").item(0)
+    languageNode = self.parser.xml.elementsByTagName("language").item(0)
+    newStateNode = self.parser.xml.createTextNode(state)
+    newGeoNode = self.parser.xml.createTextNode(geo)
+    newGeoUnmaximizedNode = self.parser.xml.createTextNode(geoUnmaximized)
+    newGeoOtherNode = self.parser.xml.createTextNode(geoOther)
+    newStateQueueNode = self.parser.xml.createTextNode(stateQueue)
+    newStateCollectorNode = self.parser.xml.createTextNode(stateCollector)
+    newStateAccountsNode = self.parser.xml.createTextNode(stateAccounts)
+    newStatePackageDockNode = self.parser.xml.createTextNode(statePackageDock)
+    newStateLinkDockNode = self.parser.xml.createTextNode(stateLinkDock)
+    newVisibilitySpeedLimitNode = self.parser.xml.createTextNode(visibilitySpeedLimit)
+    newVisibilityMaxParallelDownloadsNode = self.parser.xml.createTextNode(visibilityMaxParallelDownloads)
+    newLanguageNode = self.parser.xml.createTextNode(language)
+    stateNode.removeChild(stateNode.firstChild())
+    stateNode.appendChild(newStateNode)
+    geoNode.removeChild(geoNode.firstChild())
+    geoNode.appendChild(newGeoNode)
+    geoUnmaximizedNode.removeChild(geoUnmaximizedNode.firstChild())
+    geoUnmaximizedNode.appendChild(newGeoUnmaximizedNode)
+    geoOtherNode.removeChild(geoOtherNode.firstChild())
+    geoOtherNode.appendChild(newGeoOtherNode)
+    stateQueueNode.removeChild(stateQueueNode.firstChild())
+    stateQueueNode.appendChild(newStateQueueNode)
+    stateCollectorNode.removeChild(stateCollectorNode.firstChild())
+    stateCollectorNode.appendChild(newStateCollectorNode)
+    stateAccountsNode.removeChild(stateAccountsNode.firstChild())
+    stateAccountsNode.appendChild(newStateAccountsNode)
+    statePackageDockNode.removeChild(statePackageDockNode.firstChild())
+    statePackageDockNode.appendChild(newStatePackageDockNode)
+    stateLinkDockNode.removeChild(stateLinkDockNode.firstChild())
+    stateLinkDockNode.appendChild(newStateLinkDockNode)
+    visibilitySpeedLimitNode.removeChild(visibilitySpeedLimitNode.firstChild())
+    visibilitySpeedLimitNode.appendChild(newVisibilitySpeedLimitNode)
+    visibilityMaxParallelDownloadsNode.removeChild(visibilityMaxParallelDownloadsNode.firstChild())
+    visibilityMaxParallelDownloadsNode.appendChild(newVisibilityMaxParallelDownloadsNode)
+    languageNode.removeChild(languageNode.firstChild())
+    languageNode.appendChild(newLanguageNode)
+    self.parser.saveData()
+    self.log.debug4("main.saveWindowToConfig: done")
+
+def loadOptionsFromConfig(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [loadOptionsFromConfig is too complex (69)](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=567903)